shared/metrics/src/main/java/io/pravega/shared/MetricsTags.java

```diff
@@ -139,6 +142,13 @@ private static String getSegmentBaseName(String segmentQualifiedName) {
      */
     public static String[] createHostTag() {
         String[] hostTag = {MetricsTags.TAG_HOST, null};
+
+        //Always take property "HOSTNAME" if it's defined.
+        hostTag[1] = System.getProperty(HOSTNAME_PROPERTY_NAME);
+        if (StringUtils.isNotBlank(hostTag[1])) {
```

Comment:
Can we use `Strings.isEmpty` here? I prefer not to add a dependency from this class to micrometer just for this method.

Reply:
Yes, changed to Strings.